packages/mobile-sdk-alpha/demo-app/ios/Podfile

```
@@ -0,0 +1,36 @@
+# Resolve react_native_pods.rb with node to allow for hoisting
+require Pod::Executable.execute_command("node", ["-p",
+                                           'require.resolve(
+    "react-native/scripts/react_native_pods.rb",
+    {paths: [process.argv[1]]},
+  )', __dir__]).strip
```

RuboCop (1.76.1)

[convention] Put a comma after the last item of a multiline array.

(Style/TrailingCommaInArrayLiteral)